client/db/bolt/upgrades_test.go

```diff
@@ -18,66 +18,72 @@ import (
 	"go.etcd.io/bbolt"
 )
 
-type upgradeValidator struct {
-	upgrade upgradefunc
-	verify  func(*testing.T, *bbolt.DB)
-	expVer  uint32
-}
-
-var validators = []upgradeValidator{
-	{v1Upgrade, verifyV1Upgrade, 1},
-	{v2Upgrade, verifyV2Upgrade, 2},
-}
-
 // The indices of the archives here in outdatedDBs should match the first
 // eligible validator for the database.
 var outdatedDBs = []string{
 	"v0.db.gz", "v1.db.gz",
 }
```

Comment:
Update doc, add `"v2.db.gz"` or better still make `TestUpgradeDB` use `dbUpgradeTests` but only the `filename` property to avoid having to define filenames twice.